grow/pods/document_fields.py

```diff
@@ -0,0 +1,80 @@
+"""
+Document fields for accessing the meta fields parsed from the document.
+"""
+
+from boltons import iterutils
+import re
+
+
+LOCALIZED_KEY_REGEX = re.compile('(.*)@([^@]+)$')
+
+
+class DocumentFields(object):
```

Comment:
Doubtful - I just liked the idea of shorter file/object names. That said, encapsulating everything under `document_` sounds just fine if you think it'll be a good strategy to go with to keep things organized moving forward as we continue to work on this project.